sos/report/plugins/fibrechannel.py

```diff
@@ -18,8 +18,15 @@ class Fibrechannel(Plugin, RedHatPlugin):
     plugin_name = 'fibrechannel'
     profiles = ('hardware', 'storage', 'system')
     files = ('/sys/class/fc_host', '/sys/class/fc_remote_ports')
+    option_list = [("add_path", "additional path to collect", "fast", None)]
 
     def setup(self):
         self.add_blockdev_cmd("udevadm info -a %(dev)s", devices='fibre')
+        additional_path = self.get_option("add_path")
+        if additional_path is not None:
+            path_list = additional_path.split(',')
```

Comment:
Delimiters within plugin options have recently taken on the style of using `:` as the delimiting character. This avoids conflicts with `argparse` splitting list arguments earlier on.